README.md

```diff
@@ -834,38 +871,32 @@ no-op for the modules that already deployed successfully, and should only affect
 time around.
 
 
-
-
-
-
-
 ## Terragrunt details
 
 This section contains detailed documentation for the following aspects of Terragrunt:
 
-1. [Helper functions](#helper-functions)
+1. [Interpolation Syntax](#interpolation-syntax)
 1. [CLI options](#cli-options)
-1. [Migrating from Terragrunt v0.11.x and Terraform 0.8.x and older](#migrating-from-terragrunt-v011x-and-terraform-08x-and-older)
-1. [Terragrunt config files](#terragrunt-config-files)
+1. [Configuration](#configuration)
 1. [Developing Terragrunt](#developing-terragrunt)
 1. [License](#license)
 
 
-### Helper functions
+### Interpolation syntax
```

Comment:
Make sure there are no links in the readme to the old heading, `#helper-functions`.